### PR TITLE
Chat command options and boss command cleanup

### PR DESCRIPTION
**Pull request by devsquared**
- added BossAbbreviations
- utilized boss abbrevs for boss commands
- added options for chat commands by adding sets for each

Closes https://github.com/runelite/runelite/issues/5294

### CODE REVIEW

**Comment by devsquared** [comment]
First attempted contribution. Unsure on how to link to issue.

**Comment by devsquared** [comment]
Attempting to contribute to this issue #5294

**Comment by raiyni** [comment]
Just paste the link in your original post

Like `Closes https://github.com/runelite/runelite/issues/5294`

File: runelite-api/src/main/java/net/runelite/api/Boss.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package net.runelite.api;
+
+public enum Boss
+{
+    CORPOREALBEAST("Corporeal Beast"),
```
Comment (Kamielvf) [inline runelite-api/src/main/java/net/runelite/api/Boss.java]: Generally spaces are replaced by underscores, so all these would be e.g. `CORPOREAL_BEAST`

**Pull request closed without merging**